mlir/include/mlir/Dialect/Vector/Transforms/VectorRewritePatterns.h

```diff
@@ -364,10 +364,11 @@ void populateVectorMaskMaterializationPatterns(RewritePatternSet &patterns,
                                                PatternBenefit benefit = 1);
 
 /// Appends patterns for emulating vector operations over narrow types with ops
-/// over wider types.
+/// over wider types. `useAtomicWrites` indicates whether to use atomic
+/// operations in the places where thread contention is possible.
```

Comment:
[nit] Add a note that you are referring to MemRef atomic Ops specifically. That wasn't clear to me.